### PR TITLE
chore(rxTags): fix/refactor local midways

### PR DESCRIPTION
**Pull request by CITguy**
Angular has some strange behavior when handling the `keydown` event. This caused some inconsistent behavior when sending a `backspace` key command on the local environment, which _locally_ would cause the browser to navigate backward and cause failures with remaining midway tests.

- [x] Dev LGTM
- [x] Dev LGTM
- [x] QE LGTM

### CODE REVIEW

File: src/rxTags/rxTags.exercise.js [hunk 1]
```diff
@@ -31,37 +31,74 @@ exports.rxTags = function (options) {
...
+                before(function () {
+                    tag = component.addTag(options.sampleText);
+                });
 
-            it('removes a tag by clicking the x', function () {
-                tag.remove();
-                expect(tag.exists()).to.eventually.be.false;
-                expect(component.count()).to.eventually.equal(numTags);
-            });
+                it('should add tag as last item', function () {
```
Comment (megalord) [inline src/rxTags/rxTags.exercise.js]: The description is a little misleading, since the test doesn't check *where* the tag gets placed.

**Comment by megalord** [comment]
:+1:

File: src/rxTags/rxTags.exercise.js (hunk 1 above)
Comment (CITguy) [inline src/rxTags/rxTags.exercise.js]: updated

**Comment by bry4n** [comment]
LGTM

File: src/rxTags/rxTags.exercise.js [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+ * @exports encore.exercise.rxTags
+ * @param {Object} [options=] - Test options. Used to build valid tests.
+ * @param {rxTags} [options.instance=] - Component to Exercise.
+ * @param {string} options.sampleText - A tag that can be added.
+ * @example
+ * <pre>
+ * describe('default exercises', encore.exercise.rxTags({
+ *     instance: encore.rxTags.initialize('.demo rx-tags') // select one of many widgets on page
+ * }));
+ * </pre>
```
Comment (Droogans) [inline src/rxTags/rxTags.exercise.js]: Does this look ok in the generated documentation?

**Comment by Droogans** [comment]
Other than my one question, LGTM.

Nice job getting to the root of the issue in keydown vs. keypress.

File: src/rxTags/rxTags.exercise.js (hunk 2 above)
Comment (CITguy) [inline src/rxTags/rxTags.exercise.js]: screen shot 2015-10-05 at 6 59 11 pm

**Pull request merged**